plugin/src/test/sc/plugin2021/ManualGameTest.kts

```diff
@@ -42,7 +42,7 @@ fun loop() {
             println("$input -> $move")
             
             try {
-                game.onAction(game.gameState.getPlayer(move.color.team), move)
+                game.onAction(game.gameState.getPlayer(move.color.team)!!, move)
```

Comment:
why do we need to risk an NPE here?

Reply:
Given that we know two players already "joined" something has to be broken for `GameState.getPlayer` to return `null`.
And in case something _is_ indeed broken it is better to let the program crash

Reply:
the problem is that these exceptions are really hard to interpret. You could do a `?: throw IllegalStateException("Player missing on game start")` or something like that.

Reply:
But it isn't really an illegal state. If it was, we wouldn't have made `getPlayer()` nullable in the first place.
It is a genuine NPE - the list of players is null. Why so is a different question.
`!!` is an assertion stating that this value isn't null. If a NPE is thrown here, it should be clear immediately what the issue is (not what caused it, but at least where and what)

Reply:
tbh, `getPlayer` shouldn't be nullable. From what I've seen, in every access of it null is an illegal value, so it might as well be an exception. But I think I'll address that in a follow-up PR.

Reply:
I mostly agree.
However, it is a valid state after init before a player joined.
No one asks for the players beforehand but you technically could.